scripts/utils/generate_language_names_map.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import argparse
+import xml.etree.ElementTree as etree
+
+def extract_strings_with_id(directory):
```

Comment:
TODO: We don't need the map to include ALL language name translations, just the native translation.